repository-data/application/src/main/resources/hcm-config/namespaces/website/roadmapItem.yaml

```diff
@@ -96,6 +82,18 @@ definitions:
             jcr:primaryType: frontend:plugin
             plugin.class: org.hippoecm.frontend.editor.plugins.field.NodeFieldPlugin
             wicket.id: ${cluster.id}.field
+          /roadmapitemstats:
```

Comment:
As "roadmapitemstatus" has already been used, maybe use "roadmapitemstatuses" here (unofficial plural form :-) instead of "roadmapitemstats"

Reply:
done